src/siembol/values.yaml

```diff
@@ -0,0 +1,127 @@
+# -----------------------------------------------------------------------------
+# General seimbol parameters
+# -----------------------------------------------------------------------------
+
+# Enable TLS via certmanager
+tls:
+  enabled: true
+
+# -----------------------------------------------------------------------------
+# Alerting parameters
+# -----------------------------------------------------------------------------
+
+alerting:
```

Comment:
not needed i think

Reply:
As I mentioned, I think this job is what sets up ZooKeeper properly.  I'd love to not have it, but it doesn't feel correct have the intialization of ZooKeeper handled by the config in the `siembol-config` repo.  ZooKeeper should be initialized correctly and just work out of the box.

Reply:
to set up zk nodes - it is suffitient just create few nodes with static names and initiate them to '{}' - the rest should work out of the box

Reply:
@of13tch any thoughts on this and the conversation just above?